pkg/cli/initconfig/cmd/init.go

```diff
@@ -60,13 +59,21 @@ var initCmd = &cobra.Command{
 	Short: "init system config",
 	Long:  `init system config.`,
 	Run: func(cmd *cobra.Command, args []string) {
-		if err := run(); err != nil {
-			log.Fatal(err)
+		for {
```

Comment:
The `for` loop won't work as expected. 
It's better to wait for a while for health check and when health check passed, `initSystemConfig` could run otherwise exit.